apps/explorer/src/app/components/txs/details/tx-details-wrapper.tsx

```diff
@@ -93,6 +92,10 @@ function getTransactionComponent(txData?: BlockExplorerTransactionResult) {
       return TxDetailsNodeVote;
     case 'Withdraw':
       return TxDetailsWithdrawSubmission;
+    case 'Liquidity Provision Order':
+      return TxDetailsLiquiditySubmission;
+    case 'Amend LiquidityProvision Order':
```

Comment:
(non-blocking): if this seems ok to you
```suggestion
    case 'Amend Liquidity Provision Order':
```